asset_dashboard/models.py

```diff
@@ -250,12 +250,20 @@ class PhaseFundingYear(models.Model):
                        max_digits=11)
 
 
-class Asset(models.Model):
+class LocalAsset(models.Model):
+    """
+    We save a local copy of a geo asset with this model.
+    """
 
-    name = models.TextField()
-    location = models.GeometryField()
-    project = models.ManyToManyField(Project,
-                                     related_name='assets')
+    # TODO: implement these fields
+    phase = models.ForeignKey('Phase', on_delete=models.CASCADE)
+
+    geom = models.GeometryField(srid=3435)
+
+    # These fields correspond with the unmanaged GIS models.
+    # As far as I know, it's not possible to create a relationship
+    # between two databases with our current setup?
+    building_id = models.BigIntegerField()
```

Comment:
No, there are several asset types we'll need to accept. I was trying to implement the buildings first then add the rest of the tables. This is based off a pattern that Forest and I discussed. We were thinking that we'd have `building_id`, `trail_id`, `poi_id`, `parking_lots_id`, `picnic_grove_id`, etc, with the field holding the key that maps to the different tables in their database. I don't think that was a strict pattern to follow, but it makes sense to have those different fields so that know where the asset comes from. If we used `asset_id` or `quercus_id`, do you have any suggestion on how to keep track of the original table?

Reply:
I think we can push this into another PR, however I'd recommend this article on modeling polymorphism in Django: https://realpython.com/modeling-polymorphism-django-python/

Reply:
The `django-polymorphic` plug-in is not discussed but is also interesting. https://django-polymorphic.readthedocs.io/en/stable/quickstart.html

Reply:
@smcalilly We talked separately about storing an `asset_id` and `asset_model`.

You can get the pk field of a Django model [like this](https://stackoverflow.com/a/13419091). But it looks like you did something pretty smart with the GIS models, which is creating an `id` field on each GIS model and mapping it under the hood  to the right database column for each asset type. This gives you a unified API for querying on primary key, even though the underlying columns are different.

So, you could write a really simple property on your `LocalAsset` model to get the relevant GIS model instance, if you wanted, no complicated polymorphism required:

```python
@property
def gis_asset(self):
    # good idea to handle invalid model names and asset ids but this is an mvp
    return globals()[self.asset_model].objects.get(id=self.asset_id)
```

(`globals()` is [the current module namespace](https://docs.python.org/3/library/functions.html#globals).)